packages/entrypoint.sh

```diff
@@ -4,6 +4,11 @@ set -e
 set -u
 set -o pipefail
 
+buildmeta() {
+    set -x
+    su -l build -c "rpmspec -P ${SPEC} > ${META_DIR}/${META_NAME}"
```

Comment:
The curly braces are not needed:

```suggestion
    su -l build -c "rpmspec -P $SPEC > $META_DIR/$META_NAME"
```

Reply:
Will fix.

Reply:
You should add quote around `${META_DIR}/${META_NAME}` and `${SPEC}` in case they contains spaces no?